WindowsServerDocs/storage/storage-spaces/delimit-volume-allocation.md

```diff
@@ -28,33 +28,33 @@ Windows Server Insider Preview introduces an option to manually delimit the allo
 - Your cluster has fewer than six servers; or
 - Your cluster uses [parity](storage-spaces-fault-tolerance.md#parity) or [mirror-accelerated parity](storage-spaces-fault-tolerance.md#mirror-accelerated-parity) resiliency
 
-## Understand: Regular versus delimited allocation
+## Understand
 
-### Regular allocation
+### Review: regular allocation
 
 With regular three-way mirroring, the volume is divided into many small "slabs" that are copied three times and distributed evenly across every drive in every server in the cluster. For more details, read [this Deep Dive blog](https://blogs.technet.microsoft.com/filecab/2016/11/21/deep-dive-pool-in-spaces-direct/).
 
 ![regular-allocation](media/delimit-volume-allocation/regular-allocation.png)
```

Comment:
(Global) Your alt-text is a bit sparse. You do a good job of explaining in the text, so I don't think you need to make them super-detailed, but a little more might be good. This is pretty detailed, but I'd be tempted to write: [Diagram showing a volume divided into three stacks of slabs that are then spread uniformly across all servers.] See: [alt text guidance](https://review.docs.microsoft.com/en-us/help/style/style-how-to-alt-text?branch=master)

Reply:
Done! Thanks!